src/lib/import-export/import/validators/local-validator.ts

```diff
@@ -0,0 +1,64 @@
+import { EventEmitter } from 'events';
+import path from 'path';
+import { ImportEvents } from '../events';
+import { BackupContents } from '../types';
+import { Validator } from './validator';
+
+export class LocalValidator extends EventEmitter implements Validator {
```

Comment:
The only differences between this one and `JetpackValidator` are path prefixes and meta file names. However, I made a separate class instead of trying to have a generic one that handles both cases.